Yank/cli.py

```diff
@@ -34,6 +34,7 @@
   selftest                      Run selftests
   platforms                     List available OpenMM platforms.
   script                        Set up and run free energy calculations from a YAML script.
+  kinetics                      Initiate kinetics trajectories from checkpoint snapshots for TRAM analysis
```

Comment:
Was this meant to be included in this PR?

Reply:
Shoot, sorry. No, that shouldn't be there.

Can you create a clean PR with just the fix?

Reply:
Nevermind, I'll fix this.